fuzzers/007-timing/bel/tim2json.py

```diff
@@ -166,47 +214,54 @@ def read_raw_timings(fin, properties, pins, site_pins):
                                 elif pins[slice][site_name][delay_btype_orig][
                                         orig_pin]['direction'] == 'OUT':
                                     bel_output = pin
-                                speed_model_clean = remove_pin_from_model(
-                                    pin.lower(), speed_model_clean)
+                                if not is_alias:
```

Comment:
you check this in every case. Maybe it is worth moving this check to `remove_pin_from_model` to avoid code duplication?

otherwise LGTM

Reply:
Yes but effectively it would be calling a function "remove" with an argument which says "do not remove".

Reply:
alternatively you may change the remove function to not fail if the provided pin was not found in the bel string and simply return the original string.

I really don't like the `if not is_alias` repeated in so many places

Reply:
Ok, fixed that.